docs/vertiq_testing_tool/vertiq_testing_tool_start_guide.rst

```diff
@@ -0,0 +1,142 @@
+
+#. The :goldenrod:`Tabs` section provides different widgets for controlling your module. ESC Input provides configuration parameters necessary for integrating properly with a flight controller as well as simulating flight controller throttle commands via the IQUART raw value message. 
+   Please refer to the :ref:`Throttle Command and Modes <throttle_iquart_ref>` documentation to learn more about ESC Input.
+
+#. The :green:`Plotting` section contains a live plot in the upper section and a dashboard with widgets to control the live plot in the lower section.
+
+#. The :purple:`Control Panel` section contains widgets to configure specific parameters and to control your module. The Control Panel on each tab contains different widgets.
+
+
+.. note::
+    .. image:: ../_static/vertiq_testing_tool_pictures/module_id_widget.png
+
+    If multiple modules have been connected as described in the :ref:`Multiple Module Wiring <connection_guide>` section, you can communicate with each module by changing the Module ID in the widget shown above.
+
+==================
+Continuous command
+==================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/esc_input_command_widget.png
+
+A notable feature in the ESC Input tab is the 'Continuous command' toggle at the bottom of the Control Panel. This feature is disabled by default, which means you need to click the 'Command ESC' button every time you want to send an ESC command.
+When this feature is enabled, the ESC command will be continuously sent at the 'Command rate' with 10 Hz being the default value. The 'Command rate' can be adjusted to send the ESC command at different rates. 
+You can also use the slider to adjust the ESC input value, which mimics adjusting the flight controller's throttle input between [0, 1].
+
+
+================================
+Live Plotting and Active Metrics
+================================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input_command_example.png
+One main feature of Vertiq Testing Tool is the ability to plot any supported metric. 
+Please refer to the :ref:`IQUART Client Table Reference tables <iquart_client_reference_tables>` to find the endpoints that allow for 'get' in the Access field, 
+as these are the supported metrics that can be plotted. Also, please make sure that your module supports that client by checking its :ref:`Module Family page <module_families>`.
+
+For example, let's command the module to a 3V drive voltage using the Control Panel in the Voltage Input tab and plot the value of *drive_volts* from the Brushless Drive client.
+
+.. note::
+    Please refer to the :ref:`Velocity and Voltage Based Control Mechanisms <manual_velocity_control_mechanisms>` documentation to learn more about Velocity and Voltage Control.
+
+.. warning::
```

Comment:
The warning may want to be even before the "For example...." bit on line 87, for maximum warning in case people read that bit and start clicking buttons